modules/nw-configure-sysctl-interface-sriov-network-bonded.adoc

```diff
@@ -0,0 +1,204 @@
+// Module included in the following assemblies:
+//
+//networking/hardware_networks/configuring-sriov-device.adoc
+
+:_content-type: PROCEDURE
+[id="configure-sysctl-on-bonded-sriov-network_{context}"]
+= Configure sysctl on a bonded SR-IOV network 
```

Comment:
Before this step you need to configure sriovNetwork for bond interface like in example below:

```
apiVersion: sriovnetwork.openshift.io/v1
kind: SriovNetwork
metadata:
  name: allvalidflag 
  namespace: openshift-sriov-network-operator 
spec:
  resourceName: policyallflags
  networkNamespace: sysctl-tuning-test 
  capabilities: '{ "mac": true, "ips": true }'
```

Reply:
added configuration of sriovNetwork for bond interface